checkin/models.py

```diff
@@ -11,12 +20,30 @@ def __str__(self):
         return self.title
 
     def locale_title(self, language_code):
+        """Returns skill name in correct language.
+
+        Only 'nb' is recognised, everything else will return the english skill name.
+
+        :param language_code: Two letter string with language code
+        :return: skill name in correct language
```

Comment:
"skill name in correct language" should probably be "skill name in the given language" and capitalized.